contracts/Vault.sol

```diff
@@ -2,14 +2,22 @@
 pragma solidity ^0.6.12;
 
 import "@aave/protocol-v2/contracts/interfaces/ILendingPool.sol";
-import "@openzeppelin/contracts-ethereum-package/contracts/Initializable.sol";
-import '@openzeppelin/contracts-ethereum-package/contracts/access/Ownable.sol';
+import "@openzeppelin/contracts-ethereum-package/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/Pausable.sol";
```

Comment:
sounds good. I modified the imports to inherit from OZ's `contracts-ethereum-package` rather than contract. For some reason the integration smoke test fails with `HardhatPluginError: Error initializing integration test environment` when I try to have `MockERC20` inherit the upgradable version instead of the regular ERC20. Other tests are passing though.

Until we resolve the above we have to leave @openzeppelin/contracts in for now, but this mock contract is the last thing that depends on this package. contracts to be deployed are all inheriting from `contracts-ethereum-package`